Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN set -eux; \
 	echo '<?php phpinfo();' > /app/public/index.php
 
 COPY --link caddy/frankenphp/Caddyfile /etc/caddy/Caddyfile
-COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+ADD --chmod=0755 https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
```

Comment:
Won't this mean that every build step beyond here will be uncacheable?

Reply:
In other words, maybe we should make this one of the very last steps?

Reply:
Indeed

Reply:
Actually, it's already the last (not instant) step of this stage